internal/pkg/install/prepare.go

```diff
@@ -312,25 +328,23 @@ type Target struct {
 // Partition creates a new partition on the specified device
 // nolint: dupl, gocyclo
 func (t *Target) Partition(bd *blockdevice.BlockDevice) (err error) {
-	var (
-		typeID             string
-		legacyBIOSBootable bool
-	)
-
 	log.Printf("partitioning %s - %s\n", t.Device, t.Label)
 
 	var pt table.PartitionTable
 	if pt, err = bd.PartitionTable(true); err != nil {
 		return err
 	}
 
+	opts := []interface{}{partition.WithPartitionTest(t.Test)}
+
 	switch t.Label {
 	case constants.BootPartitionLabel:
 		// EFI System Partition
-		typeID = "C12A7328-F81F-11D2-BA4B-00A0C93EC93B"
```

Comment:
Is it worth moving these to const's right now or todo for later?

Reply:
I think I'd prefer that in another PR.